keps/sig-api-machinery/0015-dry-run.md

```diff
@@ -150,3 +151,20 @@ A dry-run request should behave as close as possible to a regular
 request. Attempting to dry-run create an existing object will result in an
 `AlreadyExists` error to be returned. Similarly, if a dry-run update is
 performed on a non-existing object, a `NotFound` error will be returned.
+
+## kubectl
+
+For the `kubectl` client integration with server-side dry-run, we will pass
+the `dryRun` query parameter by reading the user's intent from a flag.
+
+For beta, we use `--server-dry-run`. For GA, we'll use the existing `--dry-run`
```

Comment:
Currently we have `--server-dry-run` only for `kubectl apply`, and we're planning on deprecate that?

Reply:
Done.